### PR TITLE
Fix starboard toolchain when multiple are used

### PR DESCRIPTION
**Pull request by andrewsavage1**
b/376237145

### CODE REVIEW

**Comment by andrewsavage1** [comment]
Open to suggestions about how to better choose the secondary android toolchain, but it'll probably be done as a follow up

**Comment by andrewsavage1** [comment]
Unblocks @xiaomings media work

**Review by niranjanyardi — APPROVED** [review]

File: build/toolchain/android/BUILD.gn [hunk 1]
```diff
@@ -95,9 +95,17 @@ android_clang_toolchain("android_clang_x86") {
 }
 
 android_clang_toolchain("android_clang_arm") {
+  if (is_cobalt) {
+    toolchain_args = {
+      current_cpu = "arm"
+      starboard_toolchain = "//build/toolchain/android:android_clang_arm"
+      target_platform = "android-arm"
```
Comment (niranjanyardi) [inline build/toolchain/android/BUILD.gn:102]: this is overriden in args.gn file i thought , is it necessary to set it here?

Comment (andrewsavage1) [reply build/toolchain/android/BUILD.gn:102]: This is specifically for when it is used as a secondary android toolchain

**Comment by niranjanyardi** [comment]
great job fixing this , this is NOT trivial!

**Review by kaidokert — APPROVED** [review]

**Pull request merged**